plugins/vbe/fit_test.c

```diff
@@ -0,0 +1,627 @@
+/* -*- Mode: C; tab-width: 8; indent-tabs-mode: t; c-basic-offset: 8 -*-
+ *
+ * Tests for libfit
+ *
+ * Copyright (C) 2022 Google LLC
+ * Written by Simon Glass <sjg@chromium.org>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <stdio.h>
+#include <libfdt.h>
+
+#include "fu-dfu-common.h"
+
+#include "fit.h"
+#include "fit_test.h"
+
+/* Some helper macros for checking test conditions */
+#define CALL(x)		{ \
+	int iret; \
+	iret = (x); if (iret) { \
+		fprintf(stderr, "line %d: %s: %d\n", __LINE__, #x, iret); \
+		return iret; \
+	}}
+#define CHECK(x)	{ \
+	int iret; \
+	iret = (x); if (!iret) { \
+	fprintf(stderr, "line %d: %s: %d\n", __LINE__, #x, iret); \
+	return 1; \
+	}}
+#define CHECKEQ(y, x)	{ \
```

Comment:
g_assert_cmpstr please

Reply:
See above re style (don't want to require glib)